js/src/lib/components/InferenceWidget/shared/WidgetInfo/WidgetInfo.svelte

```diff
@@ -19,6 +19,19 @@
 		loaded: "This model is loaded and running on AzureML Managed Endpoint",
 		unknown: "This model can be loaded loaded on AzureML Managed Endpoint",
 	} as const;
+
+	function getStatusReport(
+		modelLoadInfo: ModelLoadInfo,
+		statuses: Record<LoadingStatus, string>
+	): string {
+		if (
+			modelLoadInfo.compute_type === "cpu" &&
+			modelLoadInfo.status === "loaded"
+		) {
+			return `The model is loaded and running on <a class="underline" href="https://huggingface.co/intel" target="_blank">Intel Xeon Ice Lake</a>.`;
+		}
```

Comment:
we could also display when it's running on GPU/TPU something else no? (in another PR)

Reply:
if I am not mistaken, currently api-interence https://github.com/huggingface/api-inference/pull/1183 outputs `compute_type` as either `cpu` or `undefined`.

I can make it show `gpu` when `compute_type` is `undefined`. Or should I ask api-interence team to explicitly provide what type of `compute_type` is ?

cc: @Narsil @OlivierDehaene

Reply:
would sound good to me